helm/blog-app/values-custom.yaml

```
@@ -0,0 +1,234 @@
+    runAsUser: 1001
+    fsGroup: 1001
+    
+  podSecurityContext:
+    seccompProfile:
+      type: RuntimeDefault
+      
+  # Environment variables
+  env:
+    NODE_ENV: production
+    PORT: 5000
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)